From: Egbert Eich <eich@suse.com>
Date: Sat Jul 10 10:43:10 2021 +0200
Subject: val_NC_check_voff(): Fix uninitialized variable warning
Patch-mainline: Not yet
Git-repo: https://github.com/Unidata/netcdf-c
Git-commit: 8b6a94698ff2a862b0188c8c74c9832c967e76f1
References: 

Make variables used in error message match the condition that triggered
the message.

Signed-off-by: Egbert Eich <eich@suse.com>
Signed-off-by: Egbert Eich <eich@suse.de>
---
 ncdump/ncvalidator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
diff --git a/ncdump/ncvalidator.c b/ncdump/ncvalidator.c
index 3cb6353f..37d7de06 100644
--- a/ncdump/ncvalidator.c
+++ b/ncdump/ncvalidator.c
@@ -2075,7 +2075,7 @@ val_NC_check_voff(NC *ncp)
 
     if (ncp->begin_rec < prev_off) {
         if (verbose) printf("Error:\n");
-        if (verbose) printf("\tRecord variable section begin offset (%lld) is less than fixed-size variable section end offset (%lld)\n", varp->begin, prev_off);
+        if (verbose) printf("\tRecord variable section begin offset (%lld) is less than fixed-size variable section end offset (%lld)\n", ncp->begin_rec, prev_off);
         nerrs++;
         DEBUG_ASSIGN_ERROR(status, NC_ENOTNC)
     }
